### PR TITLE
Drop support for 2020.2.x

### PR DESCRIPTION
**Pull request by abrooksv**
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-rider/src-203/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetLambdaHandlerResolver.kt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (manodnyab) [inline jetbrains-rider/src-203/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetLambdaHandlerResolver.kt:1]: Is there any way to detect that we've changed this other than manually?

**Review by manodnyab — APPROVED** [review]

File: jetbrains-rider/src-203/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetLambdaHandlerResolver.kt (hunk 1 above)
Comment (abrooksv) [reply jetbrains-rider/src-203/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetLambdaHandlerResolver.kt:1]: IDE updates it automatically for me

File: jetbrains-rider/tst/software/aws/toolkits/jetbrains/services/lambda/completion/DotNetHandlerCompletionTest.kt [hunk 2]
```diff
@@ -3,34 +3,26 @@
...
     override val waitForCaches = true
 
-    // TODO: Remove when https://youtrack.jetbrains.com/issue/RIDER-47995 is fixed FIX_WHEN_MIN_IS_203
-    @BeforeSuite
-    fun allowDotnetRoots() {
-        allowCustomDotnetRoots()
-    }
-
     @BeforeClass
     fun setUpBuildToolPath() {
```
Comment (rli) [inline jetbrains-rider/tst/software/aws/toolkits/jetbrains/services/lambda/completion/DotNetHandlerCompletionTest.kt:27]: what about this one?

**Review by rli — APPROVED** [review]

**Pull request merged**